lib/swarm/tracker/tracker.ex

```diff
@@ -1285,7 +1285,7 @@ defmodule Swarm.Tracker do
 
   @default_blacklist [~r/^remsh.*$/, ~r/^.+_upgrader_.+$/, ~r/^.+_maint_.+$/]
   # The list of configured ignore patterns for nodes
-  # This is only applied if no whitelist is provided.
+  # This is only applied if no blacklist is provided.
```

Comment:
I changed this from `whitelist` to `blacklist` because I am assuming it relates to `@default_blacklist`.
The third argument for [`Application.get_env/3`](https://hexdocs.pm/elixir/Application.html#get_env/3) is the default value when such configuration does not exist.